modules/kubernetes-addons/ingress-nginx/values.yaml

```diff
@@ -2,10 +2,8 @@ controller:
   service:
     externalTrafficPolicy: "Local"
     annotations:
-      # AWS Load Balancer Controller Annotations
-      service.beta.kubernetes.io/aws-load-balancer-backend-protocol: tcp # or 'ssl'
-      service.beta.kubernetes.io/aws-load-balancer-attributes: load_balancing.cross_zone.enabled=true
+      service.beta.kubernetes.io/aws-load-balancer-backend-protocol: tcp
       service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout: '60'
-      service.beta.kubernetes.io/aws-load-balancer-type: 'external'
-      service.beta.kubernetes.io/aws-load-balancer-scheme: 'internet-facing' # or 'internal'
-      service.beta.kubernetes.io/aws-load-balancer-nlb-target-type: 'ip'
+      service.beta.kubernetes.io/aws-load-balancer-cross-zone-load-balancing-enabled: 'true'
```

Comment:
This is deprecated annotation and the new one is `service.beta.kubernetes.io/aws-load-balancer-attributes: load_balancing.cross_zone.enabled=true`